make.js

```diff
@@ -228,16 +234,20 @@ target.build = function() {
                 }
             });
 
-            // npm install the common module to the task dir
-            cd(taskPath);
-            modPackFiles.forEach(function(modPackFile) {
-                run(`npm install --save-exact file://../../_build/Tasks/Common/${modPackFile}`);
-            })
+            // npm install the common modules to the task dir
+            if (modPackFiles.length) {
+                cd(taskPath);
+                var modPackPaths = modPackFiles.map(function (val) {
+                    return `file:../../_build/Tasks/Common/${val}`;
+                });
+                run(`npm install --no-save ${modPackPaths.join(' ')}`);
```

Comment:
install all the common modules into the task folder in one shot. and `--no-save` so they dont get added to the task's package.json or package-lock.json (note, the common module's bundled dependencies versions are locked down by the package-lock.json within the common module - which is what we want).